frontend/public/components/dashboards-page/overview-dashboard/inventory-card.tsx

```diff
@@ -0,0 +1,117 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+
+import * as plugins from '../../../plugins';
+import {
+  DashboardCard,
+  DashboardCardBody,
+  DashboardCardHeader,
+  DashboardCardTitle,
+} from '../../dashboard/dashboard-card';
+import { InventoryItem } from '../../dashboard/inventory-card/inventory-item';
+import { DashboardItemProps, withDashboardResources } from '../with-dashboard-resources';
+import { PodModel, NodeModel, PersistentVolumeClaimModel } from '../../../models';
+import { K8sResourceKind, PodKind } from '../../../module/k8s';
+import { getPodGroups, getNodeGroups, getPVCGroups } from '../../dashboard/inventory-card/utils';
+import { FirehoseResource } from '../../utils';
+
+const k8sResources: FirehoseResource[] = [
+  {
+    isList: true,
+    kind: PodModel.kind,
+    prop: 'pods',
+  },
+  {
+    isList: true,
+    kind: NodeModel.kind,
+    prop: 'nodes',
+  },
+  {
+    isList: true,
+    kind: PersistentVolumeClaimModel.kind,
+    prop: 'pvcs',
+  },
+];
+
+const uniqueResource = (resource: FirehoseResource, index: number): FirehoseResource => ({
+  ...resource,
+  prop: `${index}-${resource.prop}`,
+});
+
+const getResourcesToWatch = (): FirehoseResource[] => {
+  const allResources = [...k8sResources];
+  const pluginItems = plugins.registry.getDashboardsOverviewInventoryItems();
+  pluginItems.forEach((item, index) => {
+    allResources.push(uniqueResource(item.properties.resource,index));
```

Comment:
Nit: missing space after `,` (also in code below) - please run linter :smiley:

Reply:
fixed :)